spacy/gold.pxd

```diff
@@ -36,6 +36,19 @@ cdef class GoldParse:
 
     cdef readonly list cand_to_gold
     cdef readonly list gold_to_cand
-    cdef readonly list orig_annot
 
 
+cdef class RawAnnot:
+    cdef readonly list ids
+    cdef readonly list words
+    cdef readonly list tags
+    cdef readonly list heads
+    cdef readonly list deps
+    cdef readonly list ents
+    cdef readonly list brackets
+
+
+cdef class DocAnnot:
```

Comment:
I don't have a good solution, but as a set of three names, `GoldParse`, `RawAnnot` and `DocAnnot` aren't great. I'm sure we'll be able to do better.

Reply:
Yea, I agree. I started with the name "RawAnnot" to reflect the field that is stored in GoldParse with the original (non-aligned) annotations (previously called `self.orig_annot`, now just `self.orig`), but it's also used at times in the codebase where its function is more like the GoldParse itself... There's a kind of weird dependency. I think you're right that `GoldParse` should become an implementation detail.

Reply:
I think it could be something like`GoldInstance` that stores what is a document (=paragraph) in the current format. A `GoldInstance` contains a raw text, a list of the sentence/token level info as `RawAnnot`, and also some data structure that stores doc-level annotation like cats. A `GoldInstance` could provide aligned/preprocessed `GoldParses` with options to split sentences, use gold tokenization, etc. (Do you want to keep the name `GoldParse`?)